.circleci/verify-release.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
If you add `set -e`, we can avoid checking the return code below (and have the script stop execution on first error):

```suggestion
#!/bin/bash

set -e

```